README.md

```diff
@@ -12,19 +12,33 @@ The first is the proper backend code and is based on a specific image to be sure
 The second is a PostGIS DB coherent with the production for MICADO: it will install itself with all the needed schemas and configurations.
 For more details please consult the "deployment repository" [remember to create the **prod.env** file with password data]
 
+
+Prepare the backend repo on your local system:
+
+```bash
+git clone https://github.com/micado-eu/backend.git
+cd backend
+cp .env prod.env
+mkdir db_data 
+```
+
 To code you will have to run the following command on a first shell
 
 ```bash
 (set -a; source prod.env; set +a; docker-compose -f docker-compose.yaml  up backend micado_db)
 ```
+
+Wait until you see `database system is ready to accept connections`
```

Comment:
We could use wait-for-it for this step.

Reply:
I think this is a bit over kill in dev. By the time you reach that command it's probably already up and running :)